.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# Pull Request
+
+<!--- !!! PLEASE DELETE CONTENT THAT IS NOT RELEVANT !!! -->
+
+## Description
+
+<!-- A summary of the change made and how it is supposed to fix the related issue. Include relevant motivation and context. -->
+
+<!-- Fixes #(issue) -->
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Checklist:
+
+- [X] Tests are passing
+- [X] The code is properly formatted
+- [ ] The code is clean and easy to understand
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added changes to the `Unreleased` section of the CHANGELOG
```

Comment:
Add something about being even with master or having branched from master

Reply:
I didn't understand this. How is it supposed to work?

Reply:
Maybe a point like *I have created my branch from a recent version of `master`*

Reply:
why is this important?

Reply:
Ok, I got it now. It can help contributors to get updated branch before making their jobs.